WalletWasabi.Backend/CcjAnonSetTunner.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Threading;
+
+namespace WalletWasabi.Backend
+{
+	public class CcjAnonSetTunner : IDisposable
+	{
+		private static Timer _timer = new Timer(ControlRequiredAnonymitySet, null, Timeout.Infinite, Timeout.Infinite);		
+		
+		private static int completedRounds = 0;
+
+		public void Start()
+		{
+			var interval = TimeSpan.FromHours(24);
+			 _timer.Change(interval, interval);
+		}
+
+		public void Stop()
+		{
+			 _timer.Change(Timeout.Infinite, Timeout.Infinite);
+		}
+
+		public static void ControlRequiredAnonymitySet(object state)
+		{
+			if(completedRounds > Global.RoundConfig.ExpectedRoundsPerDay )
+			{
+				Global.RoundConfig.AnonymitySet = Math.Min((int)Global.RoundConfig.AnonymitySet + 1, 49);
```

Comment:
Does it work like this? Will it pick it up properly and abort existing rounds? What happens if I change the config?